tests/output.rs

```diff
@@ -339,3 +360,73 @@ fn test_auto_language_single_parseable_languages() {
         "#,
     );
 }
+
+#[test]
+fn test_capture_name() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep --query-source '(function_item name: (identifier) @name) @function_item' --language rust --capture function_item
```

Comment:
Oh, I guess I missed that you can specify the capture as an option. Is the reason for that (rather than just not capturing things you don't care about) that you might use other captures in predicates?

Reply:
Ya I guess that's probably the most common reason you'd have multiple captures? Or maybe if you were eg using a "pre-written" query?

Related-ish, I'm pretty sure that it assigns "capture indices" based on the order in which the capture names actually appear in the query (so the default would always be the capture that appears first), so here this is in fact testing something non-default (I think I ran this without the `--capture` argument to sanity-check that and did in fact not see the multiple lines corresponding to entire functions being printed out)